src/Microsoft.Identity.Client/PublicClientApplication.WithBuilders.cs

```diff
@@ -130,7 +117,11 @@ internal PublicClientApplication(ApplicationConfiguration configuration)
                 ApiEvent.ApiIds.AcquireTokenForClientWithScope, // TODO(migration): need to reconcile how to get this.  do we add this in at builder time to differentiate the various calling pattern types?
                 interactiveParameters.ExtraScopesToConsent,
                 string.IsNullOrWhiteSpace(interactiveParameters.LoginHint) ? requestParams.Account?.Username : interactiveParameters.LoginHint,
+#if NET_CORE_BUILDTIME
+                UIBehavior.SelectAccount,  // TODO(migration): fix this so we don't need the ifdef and make sure it's correct.
```

Comment:
>UIBehavior.SelectAccount [](start = 16, length = 24)

aside: I believe there has been thoughts (with evo) around the default behavior being that we do not set UIBehavior (prompt=)

Reply:
Along those lines, I think B2C doesn't want us to send anything either, they can resolve it on their end. 

---
In reply to: [247286905](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/781#discussion_r247286905) [](ancestors = 247286905)